it/src/test/java/com/linecorp/centraldogma/it/GetFileTest.java

```diff
@@ -42,34 +46,47 @@ public void testInvalidJsonPath() throws TException {
         assertThatThrownBy(() -> rule.client().getFile(
                 rule.project(), rule.repo1(), Revision.HEAD,
                 Query.ofJsonPath("/test/test2.json", "$.non_exist_path")).join())
-                .isInstanceOf(CompletionException.class).hasCauseInstanceOf(CentralDogmaException.class)
+                .isInstanceOf(CompletionException.class).hasCauseInstanceOf(
+                CentralDogmaException.class)
                 .matches(e -> ((CentralDogmaException) e.getCause()).getErrorCode() == QUERY_FAILURE);
     }
 
     @Test
-    public void testInvalidFile() throws TException {
-        assertThatThrownBy(() -> rule.client().getFile(
-                rule.project(), rule.repo1(), Revision.HEAD,
-                Query.ofJsonPath("/test/non_existing_file.json", "$.a")).join())
-                .isInstanceOf(CompletionException.class).hasCauseInstanceOf(CentralDogmaException.class)
-                .matches(e -> ((CentralDogmaException) e.getCause()).getErrorCode() == ENTRY_NOT_FOUND);
+    public void testInvalidFile() throws Exception {
+        ExpectedExceptionAppender.withExpectedException(
+                EntryNotFoundException.class, "non_existing_file", () ->
+                        assertThatThrownBy(() -> rule.client().getFile(
```

Comment:
It seems like it's fairly common to call a single `assertThatThrownBy()` with `withExpectedException`. How about adding this:

```java
import static ...ExpectedExceptionAppender.assertThatThrownByWithExpectedException;
assertThatThrownByWithExpectedException(
        EntryNotFoundException.class, "non_existing_file",
        () -> rule.client().....).isInstanceOf(...);
```

Reply:
That's a brilliant idea~! Thank you~!!